pkg/secrets/passdriver/passdriver_test.go

```diff
@@ -25,12 +25,15 @@ func setupDriver(t *testing.T) (driver *Driver, cleanup func()) {
 	})
 	require.NoError(t, err)
 
-	err = driver.gpg(context.TODO(), nil, nil, "--batch", "--passphrase", "--quick-generate-key", "testing@passdriver")
+	f, err := ioutil.TempFile("", "pass.txt")
```

Comment:
Sorry, I'm just taking a quick look. Did you mean to actually write some sort of passphrase into the file? If so have I missed where that happens?

Reply:
thanks @edsantiago I think I missed a line here. I will circle back to this today